ingestion/src/test/java/feast/ingestion/transform/ProcessAndValidateFeatureRowsTest.java

```diff
@@ -158,6 +159,56 @@ public void shouldStripVersions() {
             .setCoder(ProtoCoder.of(FeatureRow.class))
             .apply(
                 ProcessAndValidateFeatureRows.newBuilder()
+                    .setDefaultProject("myproject")
+                    .setFailureTag(FAILURE_TAG)
+                    .setSuccessTag(SUCCESS_TAG)
+                    .setFeatureSetSpecs(featureSetSpecs)
+                    .build());
+
+    PAssert.that(output.get(SUCCESS_TAG)).containsInAnyOrder(expected);
+
+    p.run();
+  }
+
+  @Test
+  public void shouldApplyDefaultProject() {
+    FeatureSetSpec fs1 =
+        FeatureSetSpec.newBuilder()
+            .setName("feature_set")
+            .setProject("myproject")
```

Comment:
Should you be providing the `project` here?

Reply:
No, it only autofills project for incoming rows, not the feature spec

Reply:
Yes, but isn't this feature spec used to create the row in the first place?

Reply:
It's manually set a few lines down

Reply:
Oops I misread.